tests/Unit/OpenApiTest.php

```diff
@@ -1,25 +1,19 @@
-<?php
-
-namespace Tests\Unit;
+<?php namespace Tests\Unit;
 
 use Descriptive\Models\Api;
 use Descriptive\Models\Response;
 use Descriptive\Models\Route;
 use PHPUnit\Framework\TestCase;
-use Symfony\Component\Yaml\Yaml;
 
 class OpenApiTest extends TestCase
 {
     /** @test */
-    public function can_create_open_api_yaml()
+    public function canCreateOpenApiYaml(): void
```

Comment:
i like snake case on tests 😄

Reply:
https://github.com/mrhn/DopeyApi/blob/293b14044d5f6a584233a19ef3b84a252396a561/.php_cs.php#L29

also more clearer approach for rules